README_KR.md

```diff
@@ -0,0 +1,110 @@
+# euphony-listener
+​
+## 소개
+​
+Euphony-Listener는 Speaker로부터 음파를 통해 데이터를 받아 인식하는 기능을 합니다. \
+스피커가 데이터를 보낼 때 주파수가 발생하게 되는데, Listen 버튼을 누르면 Listener가 데이터를 인식하여 그 결과를 확인할 수 있습니다. \
+android에서만 동작을 합니다.
+​
+## Euphony 라이브러리
+​
+해당 음파 데이터 통신 라이브러리, Euphony의 버전은 안드로이드를 위한 버전입니다.\
+Euphony는 마이크와 녹음기를 활용하여 다른 단말기(안드로이드, 웹)들과 통신하기 위해 디자인 된 무선 라이브러리를 제공합니다. 
+​
+Official Facebook Page : https://www.facebook.com/euphonyproject\
```

Comment:
![image](https://user-images.githubusercontent.com/10149398/130071728-ed756aa6-d640-4c05-ad2e-35f929344697.png)

new line is not working properly.
Instead of "\", you can also add new line with ```<space><space>```

Reply:
Is it okay to use </br>?

Reply:
Sure :) You can use <br>